http/1.5/modules/ROOT/pages/http-consume-web-svc-task.adoc

```diff
@@ -1,33 +1,96 @@
-= To Consume a REST Service (Studio)
+= Consume a REST Service Example - Mule 4
 :page-aliases: connectors::http/http-consume-web-svc-task.adoc
 
-You can design an app to consume a REST service by setting up HTTP Listener and Request operations. In this procedure, you first consume a REST API service using primarily default HTTP configurations. Next, you test the HTTP request. You edit the configuration to make a secure HTTPS request. Finally, you test the HTTPS request.
+In the following example you configure Anypoint Connector for HTTP (HTTP Connector) to consume a REST service by setting up the HTTP *Listener* and *Request* operations.
 
-. From the Mule Palette, drag an HTTP Listener operation to a new flow.
-. Set up the Listener configuration: Click + and set the default host, port and base path. For example, host = 0.0.0.0, port = 8081, and no base path.
-. In General, set Path to */trigger*.
-. Accept the other default Listener settings, and save the configuration.
-. From Mule Palette, drag an HTTP Request operation to the flow.
-. Set up the Request configuration:
+To accomplish this example, you must: +
+
+* Consume a REST API service using default HTTP configurations.
+* Test the HTTP request.
+* Edit the configuration to make secure HTTPS requests.
+* Test the HTTPS request.
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Consume a REST API Service flow
+image::http-consumerest-flow.png[Consume a REST API Service flow]
+
+== Create the Mule Application
+
+To create the Mule flow:
+
+. In the *Mule Palette* view, select the HTTP *Listener* source and drag it on to the canvas. +
+The source initiates the flow by listening for incoming HTTP message attributes.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Set the *Path* field to `/trigger` to start the app from the web browser.
```

Comment:
The Path field is not starting the app. I would leave off "to start the app from the web browser."